src/assets/sass/protocol/components/_call-out.scss

```diff
@@ -11,8 +11,9 @@
     background-color: get-theme('background-color-alt');
     text-align: center;
 
-    &.mzp-t-dark {
-        background-color: get-theme('background-color-inverse');
+    &.mzp-t-dark,
+    .mzp-t-dark & {
```

Comment:
I think we had talked about using `&.mzp-t-dark` as the standard way to theme components.

Reply:
Doh, this was a leftover from testing. Good catch.